.github/workflows/examples.yml

```diff
@@ -103,3 +105,13 @@ jobs:
         run: pnpm turbo --filter ./examples/${{ matrix.dir }} test-start
       - name: if exists, run test-dev
         run: pnpm turbo --filter ./examples/${{ matrix.dir }} test-dev
+      - name: install deno if necessary
+        if: ${{ matrix.dir === 'deno-deploy' }}
+        uses: denoland/setup-deno@v1
+        with:
+          deno-version: v1.x
+      - name: run test-dev for deno-deploy
+        if: ${{ matrix.dir == 'deno-deploy' }}
+        run: |
+          cd ./examples/${{ matrix.dir }}
+          pnpm start-server-and-test 'deno run --allow-net=:8000 --allow-env ./src/index.ts' http://127.0.0.1:8000 'deno run --allow-net ./src/client.ts'
```

Comment:
This will run the deno test this many times:

https://github.com/trpc/trpc/blob/1030b0e1ce59cc90e4bbbb5e36ef1151b69f3017/.github/workflows/examples.yml#L24-L40


Instead, do a new `jobs: deno` at the bottom with a new matrix

Reply:
I'm filtering with an `if` to only run on the `deno-deploy` `matrix.dir`, so it should run only once. Should I still add a new job instead?

Reply:
Ahhh, yes, I missed that if


but do it in it's own job instead so it's clean separation between node and deno